### ISSUE TITLE
Cuneiform ML Translations

### ISSUE THREAD

**Issue by Ur-Namma** [praeclarum/CuneiformTranslators#4]
What you have done is incredible.  You are 100% right that most translations are, in fact, transliterations and, as such, the transliterations regularly impute a great deal of modern bias (consciously or otherwise) and questionable contextual framing  on such texts.  What has emerged in many cases is a hierarchy of interpretations on interpretations, a great deal of which almost certainly obfuscates insights that can only be de-tangled by starting from as literal translations as possible.

Have you considered including Hittite, Kassite, or other spoken languages rendered in cuneiform as part of your dataset?  A great number of such tablets are known, most of which have not been translated (many are held in Turkey)?  

There is also potential to go one profoundly big step forward.  Prior to the use of cuneiform, there appears to have been wide-scale use of logographic scripts which share remarkable similarities in the recurring use of certain glyphs globally (that is to say, many identical bodies of glyphs appear in regions as diverse as early Bronze age Minoan Crete (Linear-A), Shang Dynasty (and earlier) bird-worm and similar scripts in what is now China and the Khmer/northing mountainous regions of Thailand and Vietnam, various pre-Buddhist scripts from oceanic SE Asia, Egyptian hieroglyphs, 'proto-Elamite, etc..  This is the focus of my research and I have assembled a pretty decent data-set of such scripts, and a great number of such examples are also in the digital archives of the data-sets you used for the cuneiform translations (catalogued under 'pre-writing).  I'd estimate a few thousand digitized examples exists and since the field of linguistics largely concentrates on alphabetical scripts, the insights from even a rough ML analysis of these scripts could be profoundly important.  Many were in use since literally the dawn of human memory, and a recurring, identical body of said glyphs appear in the highly sophisticated cave art of modern day Southern-France, Northern Iberia, and Sulawesi, Indonesia, to name just a few, dating to 30-41,000 years ago.

**Comment by praeclarum** [comment]
Yes, I absolutely do plan on supporting more languages with Hittite being the most obvious next one (due to large amounts of data). In fact, the 1st edition of these translations had a variety of languages including some forms of Hittite and Elamite.

I am very interested in expanding the cuneiform to earlier scripts as I've found (obviously in retrospect) that the translations improve when I inform the network of rough eras so it can tune its understanding. I have also really wanted to branch into image recognition so we can skip the whole transliteration step.

I apologize for the slow delay but I would be very interested in seeing what data you have and judging whether I could provide any help.

Have a great day and I hope to talk more.

**Comment by Ur-Namma** [comment]
Sorry if redundant, fat fingered last attempt.  To keep this brief, cuneiform was always logographic first, and the generally poor and inconsistent quality of transliterations is a produce of a modern attempt to impute alphabetical/syllabic elements on the glyphs.  It was always spoken language independent, and used for Hittite, Kassite, Akkadian, early Elamite, and others.  For many of these, the earlier logographic/hieroglyphic scripts still exist, so you have both, sole well cataloged, which is where your code can be super powerful.  That is, almost none of the earlier hieroglyphics are deciphered/translated, and where they have been (like Egyptian), they too suffer from this alphabetical/syllabic bias.  What is very clear (you can see it with your two eyes very quickly) is both the recurrence of common glyphs in nearly all the known hieroglyphic scripts, which tend to recur in context (they often appear at specific places and in association with other glyphs in the texts, which is non-random).  A few example are the various anatolian hieroglyphs (Lewian,  Hittite, etc), Minoan Cretan, Egyptian, and even early Chinese bird-worm/seal scripts, Mesoamerican hieroglyphs, and Rongo Rongo (Easter Island).  So with the cuneiform (in the case of Elamite and others), well catalogued logographic datasets, and the other layers syllabic/alphabetical scripts, there is probably plenty of good learning data to make inroads on the hard problems of deciphering the symbolic languages, which almost certainly share some much earlier common symbolic substrate writing.  The point is, I doubt any of these will be deciphered through a vector of alphabetizing them, but your code can help find common features and contribute to a more logographic approach.  

Let me know if you want more specifics, but here's an excellent archive of many such scripts:  https://www.unicode.org/charts/.

**Comment by Weirdvideos** [comment]
Dear praeclarum;   On your web site  https://aicuneiform.com/,  how can I download the entire collection into one diretory file, for reading all WITHOUT clicking from one page to another?  I would use my AI to scan and compile subject documentations.  I would die of old age first before I could read all the pages.  PLEASE reply?  ROSS S Mardshall,  < rossmarshall@weirdvideos.com > 1-360-421-7195 WA. USA.
